index.js

```diff
@@ -1,6 +1,12 @@
 'use-strict'
 
 module.exports = {
+
+  Controller: require('./lib/controllerFactory'),
```

Comment:
why?

Reply:
seem to have been caused due to a rebase error...resolving them now..

Reply:
created new PR https://github.com/coding-blocks/express-jsonapi-controller/pull/9